tests/test.xpi.js

```diff
@@ -0,0 +1,238 @@
+import Xpi from 'xpi';
+import { Readable } from 'stream';
+
+
+const defaultData = {
+  compressionMethod: 8,
```

Comment:
Defining those numbers as constants in our Xpi class and using them (`Xpi.COMPRESS_METHOD_DEFLATE` or whatnot) would work then. A comment is also fine if you find that too verbose, but I like to name magic constants like this.

- tofumatt

> On 25 Sep 2015, at 09:10, Stuart Colville <notifications@github.com> wrote:
> 
> In tests/test.xpi.js:
> 
> > @@ -0,0 +1,238 @@
> > +import Xpi from 'xpi';
> > +import { Readable } from 'stream';
> > +
> > +
> > +const defaultData = {
> > +  compressionMethod: 8,
> This is based on a sample of the data we get back from yauzl. So we're not defining that. But 8 means deflate, 0 means no compression. I left this is because I think we'll probably want to explode if it's not 8 or 0 as they are the only supported values.
> 
> —
> Reply to this email directly or view it on GitHub.
>